contracts/truefi2/DebtToken.sol

```diff
@@ -75,10 +72,10 @@ contract DebtToken is IDebtToken, ERC20 {
      * @dev Liquidate the debt if it has defaulted
      */
     function liquidate() external override {
-        require(status == Status.Defaulted, "DebtToken: Current status should be Defaulted");
+        require(!hasLiquidated, "DebtToken: Debt must not be liquidated");
```

Comment:
I'd prefer `isLiquidated`

Reply:
+1, that's how we do it in our code everywhere